app/src/main/java/com/vitorpamplona/amethyst/ui/screen/loggedIn/AccountBackupDialog.kt

```diff
@@ -121,3 +121,77 @@ private fun NSecCopyButton(
         Text("Copy Secret Key", color = MaterialTheme.colors.onPrimary)
     }
 }
+
+fun Context.getFragmentActivity(): FragmentActivity? {
+    var currentContext = this
+    while (currentContext is ContextWrapper) {
+        if (currentContext is FragmentActivity) {
+            return currentContext
+        }
+        currentContext = currentContext.baseContext
+    }
+    return null
+}

```

Comment:
We need to unwrap the wrapped composable context to get the fragment back, since the Biometrics accepts no other type of context.